### PR TITLE
Addition of package rsnapshot.

### PR DESCRIPTION
**Pull request by ketandixit**
This adds the rsnapshot package. It has scripts to download, compile and install rsnapshot

Fixes #ENGT-6930

@earlruby, @bwerthmann, @variadico

### CODE REVIEW

File: packages/rsnapshot-1.4.2.conf [hunk 1]
```diff
@@ -0,0 +1,37 @@
+name:      "rsnapshot-1.4.2"
+namespace: "/apcera/pkg/packages"
+
```
Comment (earlruby) [inline packages/rsnapshot-1.4.2.conf]: Some packages also have a `version:` tag, e.g. `version: "1.4.2"`, although I haven't seen it documented anywhere.

File: packages/rsnapshot-1.4.2.conf [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+sources [
+  { url: "http://rsnapshot.org/downloads/rsnapshot-1.4.2.tar.gz",
+    sha256: "042a81c45b325296b21c363f417985d857f083f87c5c27f5a64677a052f24e16" }
```
Comment (earlruby) [inline packages/rsnapshot-1.4.2.conf]: You should use `contempt` to store a copy of `rsnapshot-1.4.2.tar.gz` in our s3 "sources" bucket. See https://github.com/apcera/contempt#upload

**Comment by earlruby** [comment]
LGTM after squash and rebase.

**Comment by variadico** [comment]
LGTM after squash and rebase.

**Pull request merged**